src/query/ts/metadata.go

```diff
@@ -47,16 +84,15 @@ const (
 
 // SeriesAttributes has attributes about the time series.
 type SeriesAttributes struct {
-	Type   MetricType
-	Source SourceType
+	M3Type         M3MetricType
+	PromType       PromMetricType
+	PromFamilyType PromMetricFamilyType
```

Comment:
Can we split this into two types? One which covers the input prom types, and one that is converted into that only describes the m3 type? Otherwise there's potential situations where these may drift

Reply:
This is changing now that I am removing the `FamilyType` concept (as per our earlier discussion). With that in mind, can you clarify what exactly do you want to split (the comment is not very clear to me)?